core/src/main/scala/cell/HandlerPool.scala

```diff
@@ -29,6 +26,35 @@ class HandlerPool(parallelism: Int = 8, unhandledExceptionHandler: Throwable =>
 
   private val cellsNotDone = new AtomicReference[Map[Cell[_, _], Cell[_, _]]](Map())
 
+  /**
+   * Returns a new cell in this HandlerPool.
+   *
+   * Creates a new cell with the given key. The `init` method is used to
+   * retrieve an initial value for that cell and to set up dependencies via `whenNext`.
+   * It gets called, when the cell is awaited, either directly by the awaitResult method
+   * of the HandlerPool or if a cell that depends on this cell is awaited.
+   *
+   * @param key The key to resolve this cell if in a cycle or insufficient input.
+   * @param init A callback to return the initial value for this cell and to set up dependencies.
+   * @param lattice The lattice of which the values of this cell are taken from.
+   * @return Returns a cell.
+   */
+  def createCell[K <: Key[V], V](key: K, init: () => Outcome[V])(implicit lattice: Lattice[V]): Cell[K, V] = {
```

Comment:
The convention in Scala's core library is to use "mk" as the prefix, so let's call it "mkCell".